.tekton/open-infra-deployment-pr-osp-nightly.yaml

```diff
@@ -18,8 +18,12 @@ spec:
     - name: infra-deployment-update-script
       value: |
         sed -i -E 's/[0-9a-f]{40}/{{ revision }}/g' components/pipeline-service/development/kustomization.yaml
+        # just pin the watcher image, which is the second image ref, so use 2 vs. g
+        sed -i -E 's/newTag: {{ revision }}/newTag: bae7851ff584423503af324200f52cd28ca99116/2' components/pipeline-service/development/kustomization.yaml
         sed -i -E 's/[0-9a-f]{40}/{{ revision }}/g' components/pipeline-service/staging/base/kustomization.yaml
-        sed -i -E 's/[0-9a-f]{40}/{{ revision }}/g' components/monitoring/grafana/base/dashboards/pipeline-service/kustomization.yaml
+        # just pin the watcher image, which is the second image ref, so use 2 vs. g
+        sed -i -E 's/newTag: {{ revision }}/newTag: bae7851ff584423503af324200f52cd28ca99116/g' components/pipeline-service/staging/base/kustomization.yaml
```

Comment:
Based on the comment you intended to use `/2` but left it `/g`. Also see my previous comment.